packages/analytics/metric-cards/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "devDependencies": {
     "@kong/design-tokens": "^1.11.6",
+    "@kong/icons": "^1.7.8",
```

Comment:
Just to double-check... I would have guessed this would be a `peerDependency` -- why is it a dev dependency?

Reply:
Good question. I just did what @adamdehaven told me to do, which is install it as a devDependency :)

Now that I look at the rest of `public-ui-components`, `@kong-icons` is used under `dependencies` in 2 other places:

https://github.com/Kong/public-ui-components/blob/main/packages/core/error-boundary/package.json#L65
https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-shared/package.json#L72

Reply:
you need it as an actual dependency since you're using the rendered SVG; technically it _could_ be a peerDep, but the package is so light it doesn't really matter TBH